cliclient/cliclient.go

```diff
@@ -148,18 +149,21 @@ func (mc *MasterClient) ByID(resource *ntypes.Resource, respObject interface{})
 		if err != nil {
 			return err
 		}
+		return nil
```

Comment:
maybe `return mc.ManagementClient.ByID(resource.Type, resource.ID, &respObject)` looks better?

Reply:
Changed it!